src/Elmish.WPF/BindingData.fs

```diff
@@ -541,6 +474,22 @@ module internal BindingData =
 
   module SubModel =
 
+    let mapMinorTypes
+        (outMapBindingModel: 'bindingModel -> 'bindingModel0)
+        (outMapBindingMsg: 'bindingMsg -> 'bindingMsg0)
+        (outMapBindingViewModel: 'bindingViewModel -> 'bindingViewModel0)
+        (inMapBindingModel: 'bindingModel0 -> 'bindingModel)
+        (inMapBindingMsg: 'bindingMsg0 -> 'bindingMsg)
+        (inMapBindingViewModel: 'bindingViewModel0 -> 'bindingViewModel)
+        (d: SubModelData<'model, 'msg, 'bindingModel, 'bindingMsg, 'bindingViewModel>) = {
+      GetModel = d.GetModel >> ValueOption.map outMapBindingModel
+      CreateViewModel = fun args -> d.CreateViewModel(args |> ViewModelArgs.map inMapBindingModel outMapBindingMsg) |> outMapBindingViewModel
```

Comment:
```suggestion
      CreateViewModel = fun args -> args |> ViewModelArgs.map inMapBindingModel outMapBindingMsg |> d.CreateViewModel |> outMapBindingViewModel
```

Reply:
Or even
```suggestion
      CreateViewModel = ViewModelArgs.map inMapBindingModel outMapBindingMsg >> d.CreateViewModel >> outMapBindingViewModel
```

Reply:
I prefer having `d.CreateViewModel` first. Also this change doesn't actually modify this line, it just moves it.